### PR TITLE
More Misc Tweaks - Flow / Consistency Improvements

### PR DESCRIPTION
**Pull request by janoside**
This is another dump of misc tweaks in the same spirit as PR #62, gathered as I finished my read-through.

### CODE REVIEW

**Comment by mflaxman** [comment]
Thanks @janoside! There are merge conflicts and also the spellcheck fails (see `.wordlist.txt`), would you like to make changes so it is mergeable?

**Comment by janoside** [comment]
@mflaxman You're welcome - thanks for all your hard work on this guide. I think everything has been cleaned up and is mergeable now.

File: _pages/emergency-recovery/advanced.md [hunk 1]
```diff
@@ -17,3 +17,6 @@ Tails strongly recommended.
 
 #### Verify Validity of Electrum Software
 You can read instructions on how to do this [here](https://www.reddit.com/r/Bitcoin/comments/aa3l6n/how_to_check_if_your_electrum_wallet_is_legit/).
+
+
+{% include next_steps.md %}
```
Comment (mflaxman) [inline _pages/emergency-recovery/advanced.md:22]: Why did you add this? As-is it does nothing. It's the end of the guide, maybe next steps is back to table of contents?

File: _pages/known-issues/hardware/specter-diy.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ TODO: add more content
 {% include hw/experts.md %}
 
 #### Cannot Buy Assembled Version
-While the DIY version is great for expert users (and no longer requires soldiering!), the overwhelming majority of users prefer a product they can purchase.
+While the DIY version is great for expert users (and no longer requires soldering!), the overwhelming majority of users prefer a product they can purchase.
```
Comment (mflaxman) [inline _pages/known-issues/hardware/specter-diy.md:10]: Lol

File: _pages/known-issues/hosted/unchained.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ As Trezor is stateless, these steps will only confirm that `1` of your keys is p
  [`3-of-5` would be better](/quorum-advanced), especially when using a hosted service that already:
 
 * Abstracts away a lot of multisig's complexity
-* Holds `1` (or more) of your seeds
+* Holds 1 (or more) of your seeds
```
Comment (mflaxman) [inline _pages/known-issues/hosted/unchained.md:19]: The convention in the guide has been to use `code` formatting for `m-of-n`, do you think that's bad? I like it, but consistency is probably the most important. IThere are other examples that show it as `1` above on this very page.

File: _pages/known-issues/verify-receive-address.md [hunk 4]
```diff
@@ -12,8 +12,8 @@ Let's say you have a `2-of-3` multisig (with seeds A, B, and C), and your host c
 Unfortunately for you, one of your hardware wallets is compromised/fake.
 Here is how you could be tricked into using an address you don't actually control to receive funds:
 
-1. Specter on your host computer (malware) displays a `2-of-3` receiving address, where *none* of your hardware wallets are actually part of the multisig scheme.
-1. You pull out hardware wallet A (also malware!) and it (falsely) confirms the same receive address on its trusted display.
+1. Specter-Desktop on your host computer (assumed to be malware infected) displays a `2-of-3` receiving address, where *none* of your hardware wallets are actually part of the multisig scheme.
+1. You pull out hardware wallet A (also malware infected!) and it (falsely) confirms the same receive address on its trusted display.
```
Comment (mflaxman) [inline _pages/known-issues/verify-receive-address.md:16]: This is not your fault, but what I wrote here is confusing. I'm going to rework this.

File: _pages/setup-wallets/coldcard-advanced.md [hunk 5]
```diff
@@ -35,9 +36,12 @@ In order to verify a receive address on a coldcard you must significantly weaken
...
-Expert users can figure out how to create a multisig wallet on their coldcard with only 1 file from their hot machine (no need to first export the public key information from the coldcard).
+Each time an SD card is taken from your computer to your Coldcard it can theoretically introduce malware onto the Coldcard (a failure where the malware is said to have "jumped your airgap").
+Expert users can figure out how to create a multisig wallet on their Coldcard with only 1 file from their hot machine (no need to first export the public key information from the Coldcard).
 
 ## Not Perfect
 No device is perfect and this one is no exception.
 Read more about known issues with Coldcard [here](/known-issues/hardware/coldcard).
+
+
+{% include next_steps.md next_url="coordinate-multisig" next_name="Coordinate Multisig" %}
```
Comment (mflaxman) [inline _pages/setup-wallets/coldcard-advanced.md:47]: Good catch

File: _pages/setup-wallets/paper-advanced.md [hunk 6]
```diff
@@ -111,3 +111,6 @@ Be sure to use a hardware wallet where you can enter the passphrase directly on
 ## Not Perfect
 No setup is perfect and this one is no exception.
 Read more about known issues with SeedPicker [here](/known-issues/software/seedpicker).
+
+
+{% include next_steps.md next_url="cobo" next_name="Setup Cobo Vault" %}
```
Comment (mflaxman) [inline _pages/setup-wallets/paper-advanced.md:116]: Good catch

**Review by mflaxman — APPROVED** [review]
This is so good (thanks!) I'm going to merge now. If you want to cleanup some of the nitpick items feel free to submit another PR, or else I will clean them up eventually.

Would you like to be mentioned as a contributor?

File: _pages/emergency-recovery/advanced.md (hunk 1 above)
Comment (janoside) [reply _pages/emergency-recovery/advanced.md:22]: This was just copied from `/emergency-recovery/index.md`. Copying from the related "non-advanced" page is what I did for the other missing "next steps" and I just did the same here, not realizing it didn't really apply!

**Comment by janoside** [comment]
I'm happy to help! No need for contributor call-out, but thank you.

File: _pages/known-issues/hosted/unchained.md (hunk 3 above)
Comment (janoside) [reply _pages/known-issues/hosted/unchained.md:19]: I do like the `code` formatting for quorum descriptions (e.g. `2-of-3`). I'm on the fence about using it for individual numbers but, regardless, I agree consistency is more important and that convention is used elsewhere (I just didn't register the few other uses).

PR incoming...
